# Info message about current battery status

Header header

float32 ActualCurrent_A   # in A
float32 Voltage_V         # in V
float32 UsedCapacity_mAh  # in mAh


